opentelemetry-user-events-metrics/src/exporter/mod.rs

```diff
@@ -0,0 +1,87 @@
+use crate::transform::transform::transform_resource_metrics;
+use async_trait::async_trait;
+
+use opentelemetry_api::metrics::{MetricsError, Result};
+use opentelemetry_sdk::{
+    metrics::{
+        data::{ResourceMetrics, Temporality},
+        exporter::PushMetricsExporter,
+        reader::{
+            AggregationSelector,
+            TemporalitySelector,
+        },
+        Aggregation, InstrumentKind,
+    },
+};
+
+use std::fmt::{Debug, Formatter};
+use prost::Message;
+use crate::tracepoint;
+
+#[derive(Clone, Copy)]
+pub struct MetricsExporter { }
+
+impl MetricsExporter {
+
+    pub fn new() -> MetricsExporter {
+        // This is unsafe because if the code is used in a shared object (DLL),
+        // the event MUST be unregistered before the shared object unloads.
+        unsafe {
+            let result = tracepoint::register();
+            if result != 0 {
+                println!("Tracepoint failed to register.");
+            }
+        }
+        MetricsExporter {  }
+    }
+}
+
+impl TemporalitySelector for MetricsExporter {
+    fn temporality(&self, _kind: InstrumentKind) -> Temporality {
+        Temporality::Cumulative
```

Comment:
please add TODO here as these should be not just Cumulative

Reply:
This seems to be the default temporality, can we have `Delta` as default ?

Reply:
Default temporality will probably be implemented with a separate "DefaultTemporalitySelector" (e.g. [DefaultTemporalitySelector](https://github.com/open-telemetry/opentelemetry-rust/blob/a08c6406bb8eadcb8568740a377cfce913a1b66f/opentelemetry-sdk/src/metrics/reader.rs#L99) when we decide to implement this. As for now, I will leave a TODO.